app/cli/pkg/token.go

```diff
@@ -0,0 +1,99 @@
+//
+// Copyright 2024-2025 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package token
```

Comment:
I believe this is not how packages work in golang, everything in `pkg` must be the same package, so with your setup, you can only have `token` in the `pkg/` directory, is that what you wanted?

If that's not what you wanted, I'd suggest to do

`cli/internal/token/token.go`, internal so it "indicates" it's not for sharing with other apps, and `token/` to create the actual package namespace. Does it make sense?

Reply:
btw, just FYI, I find this reference guide quite handy https://go.dev/doc/effective_go#package-names